python/desc/twinkles/twinklesDBConnections.py

```diff
@@ -38,3 +39,52 @@ class SNCacheDBObj(SNDBObj, CatalogDBObject):
 
     def query_columns(self, *args, **kwargs):
         return CatalogDBObject.query_columns(self, *args, **kwargs)
+
+class TwinklesBulgeObj(GalaxyBulgeObj):
+
+    database = 'LSSTCATSIM'
```

Comment:
Should we just define these connection parameters in a mixin that all of the `Twinkles*Obj` classes can inherit from?